gateway-contracts/contracts/Decryption.sol

```diff
@@ -677,39 +682,39 @@ contract Decryption is
     /// @param userDecryptRequestVerification The UserDecryptRequestVerification structure to hash.
```

Comment:
Maybe we'd like to include the `/// @param hostChainId ...`

Same in `_hashDelegatedUserDecryptRequestVerification` method.